### PR TITLE
Be-update-leaguetable

### PR DESCRIPTION
**Pull request by Stephen0wen**
Updates leaguetable endpoint so that it includes teams with no completed fixtures

### CODE REVIEW

File: back-end/models/league_tables.model.js [hunk 1]
```diff
@@ -1,5 +1,28 @@
...
+        team2.team_name AS team2_name,
+        team1_score,
+        team2_score,
+        team1.team_division AS division
+      FROM fixtures
+      JOIN teams AS team1
+        ON fixtures.team1_id = team1.team_id
+      JOIN teams AS team2
+        ON fixtures.team2_id = team2.team_id
+      WHERE fixtures.match_status = 'completed'
```
Comment (miors) [inline back-end/models/league_tables.model.js:18]: Say a team called XYZ has never completed any match, shouldn't the query also capture it and subsequently display it in the league table?

**Review by miors — COMMENTED** [review]
Looks good, only a question to reconfirm

File: back-end/models/league_tables.model.js (hunk 1 above)
Comment (Stephen0wen) [reply back-end/models/league_tables.model.js:18]: Indeed it should (and does). This is achieved by a second query call in the controller (reusing the teams model). The util function creates a zero score league table row for every team and then iterates through the fixtures to add the necessary valued.

Comment (Stephen0wen) [reply back-end/models/league_tables.model.js:18]: If you make a get request in insomnia (or just in your browser as it is only a GET request) you should see teams who haven't played any games in the response

Comment (miors) [reply back-end/models/league_tables.model.js:18]: I understand now. I just tested it and could confirm I observed the correct results. Great work @Stephen0wen 👍

**Review by miors — APPROVED** [review]
LGTM 🎉

**Pull request merged**